bin/import_cv_personal.py

```diff
@@ -0,0 +1,219 @@
+    if os.path.exists(wav_filename):
+        file_size = os.path.getsize(wav_filename)
+        frames = int(
+            subprocess.check_output(
+                ["soxi", "-s", wav_filename], stderr=subprocess.STDOUT
+            )
+        )
+    label = FILTER_OBJ.filter(sample[1])
+    rows = []
+    counter = get_counter()
+    if file_size == -1:
+        # Excluding samples that failed upon conversion
+        counter["failed"] += 1
+    elif label is None:
+        # Excluding samples that failed on label validation
+        counter["invalid_label"] += 1
+    elif int(frames / SAMPLE_RATE * 1000 / 10 / 2) < len(str(label)):
+        # Excluding samples that are too short to fit the transcript
+        counter["too_short"] += 1
+    elif frames / SAMPLE_RATE > MAX_SECS:
+        # Excluding very long samples to keep a reasonable batch-size
+        counter["too_long"] += 1
+    else:
+        # This one is good - keep it for the target CSV
+        rows.append((os.path.split(wav_filename)[-1], file_size, label, sample[2]))
+        counter["imported_time"] += frames
+    counter["all"] += 1
+    counter["total_time"] += frames
+
+    return (counter, rows)
+
+
+def _maybe_convert_set(tsv_file, audio_dir, space_after_every_character=None):
+    rows = []
+    input_tsv = os.path.abspath(tsv_file)
+    print("Loading TSV file: ", input_tsv)
+    # Get audiofile path and transcript for each sentence in tsv
+    samples = []
+    with open(input_tsv, encoding="utf-8") as input_tsv_file:
+        reader = csv.DictReader(input_tsv_file, delimiter="\t")
+        for row in reader:
+            samples.append(
+                (
+                    os.path.join(audio_dir, row["original_sentence_id"] + ".mp3"),
+                    row["sentence"],
+                    row["locale"],
+                )
+            )
+    counter = get_counter()
+    num_samples = len(samples)
+    print("Importing mp3 files...")
+    pool = Pool(initializer=init_worker, initargs=(PARAMS,))
+    bar = progressbar.ProgressBar(max_value=num_samples, widgets=SIMPLE_BAR)
+    for i, processed in enumerate(pool.imap_unordered(one_sample, samples), start=1):
+        counter += processed[0]
+        rows += processed[1]
+        bar.update(i)
+    bar.update(num_samples)
+    pool.close()
+    pool.join()
+
+    imported_samples = get_imported_samples(counter)
+    assert counter["all"] == num_samples
+    assert len(rows) == imported_samples
+    print_import_report(counter, SAMPLE_RATE, MAX_SECS)
+
+    output_csv = os.path.join(os.path.abspath(audio_dir), "data.csv")
+    print("Saving new Coqui STT-formatted CSV file to: ", output_csv)
+    with open(output_csv, "w", encoding="utf-8", newline="") as output_csv_file:
+        print("Writing CSV file for train.py as: ", output_csv)
+        writer = csv.DictWriter(output_csv_file, fieldnames=FIELDNAMES)
+        writer.writeheader()
+        bar = progressbar.ProgressBar(max_value=len(rows), widgets=SIMPLE_BAR)
```

Comment:
nit: Is this really necessary? Who's gonna have a personal export so large that just writing some rows into a file is gonna take long enough to need a progress bar?